tests/client/amo/components/TestFeaturedAddons.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import * as featuredActions from 'amo/actions/featured';
+import {
+  FeaturedAddonsBase,
+  mapStateToProps,
+} from 'amo/components/FeaturedAddons';
+import createStore from 'amo/store';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import { fakeAddon } from 'tests/client/amo/helpers';
+import { getFakeI18nInst, shallowRender } from 'tests/client/helpers';
+
+
+describe('<FeaturedAddons />', () => {
+  const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
+
+  function render({ ...props }) {
+    return shallowRender(
+      <FeaturedAddonsBase i18n={getFakeI18nInst()} {...props} />
```

Comment:
Because I'm using `shallowRender` (to avoid having to add `<Provider>`s and render a bunch of children), that seems to not work. I get errors like:

```
✖ renders a FeaturedAddons page with themes HTML
      Firefox 50.0.0 (Mac OS X 10.12.0)
    root.props.children is undefined
    @webpack:///tests/client/amo/components/TestFeaturedAddons.js:44:4 <- test-runner.js:126801:6
```